### DISCUSSION TITLE
Awesome helper scripts

### DISCUSSION THREAD

**Discussion by Knusper** [neomutt/neomutt#4373] [Development]
A year ago or so, I discovered these awesome tools for use with mutt / neomutt:

https://hg.phloxic.productions/muttils/file/c4d11634661f/README

I don't believe that they are widely known, but they really make life better.  Perhaps these could be mentioned in the documentation or so?

**Comment by Knusper** [comment]
My current affiliation uses some software from cisco that parses incoming mail and uses an external redirect for all links.
Unfortunately, this obfuscates the links. 
There is  a nice script, that can be easily integrated with neomutt to de-obfuscate such links: 
https://www.sthu.org/code/codesnippets/mutt-cisco-url-rewriting-clean.html

**Comment by flatcap** [comment]
Nice!

We only mirror a few [external contributions](https://github.com/neomutt/neomutt/tree/main/contrib) in the **code repo**.
However, you're welcome to add as much as you like to the [Useful Programs](https://neomutt.org/contrib/useful-programs) in the **website**.

Thanks!

**Reply by Knusper** [reply]
created a PR to add the things to the website

**Comment by flatcap** [comment]
Merged in https://github.com/neomutt/neomutt.github.io/commit/003ec4b96a708066fc93af34ee9db590880cfc13
